src/main/java/macrobase/diagnostics/ScoreDumper.java

```diff
@@ -0,0 +1,41 @@
+package macrobase.diagnostics;
```

Comment:
@edgan8 talked about having a scoredump operator that you could stick into the pipeline. what do you think about that?